<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estimatex</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="Name_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Formula_bar"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="TabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImport_xlm"/>
     <addaction name="actionImport_xlsx"/>
    </widget>
    <widget class="QMenu" name="menuEksport">
     <property name="title">
      <string>Eksport</string>
     </property>
     <addaction name="actionExport_xlsx"/>
     <addaction name="actionExport_pdf"/>
     <addaction name="actionExport_xml"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuEksport"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>Nowy...</string>
   </property>
   <property name="iconText">
    <string>Nowy kosztorys</string>
   </property>
   <property name="toolTip">
    <string>Nowy kosztorys</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Otwórz...</string>
   </property>
   <property name="iconText">
    <string>Otwórz kosztorys</string>
   </property>
   <property name="toolTip">
    <string>Otwórz kosztorys</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Zapisz...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Zapisz jako...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Zamknij</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionImport_xlm">
   <property name="text">
    <string>Kosztorys (*.xml)</string>
   </property>
  </action>
  <action name="actionExport_xlsx">
   <property name="text">
    <string>Plik XLSX</string>
   </property>
  </action>
  <action name="actionImportCsv">
   <property name="text">
    <string>*.csv</string>
   </property>
  </action>
  <action name="actionExport_pdf">
   <property name="text">
    <string>Plik PDF</string>
   </property>
  </action>
  <action name="actionExport_xml">
   <property name="text">
    <string>Plik xml</string>
   </property>
  </action>
  <action name="actionImport_xlsx">
   <property name="text">
    <string>.XLSX</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>resources.TabWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
